subscriptions/tasks.py

```diff
@@ -122,6 +129,19 @@ def _charge_recurring_subscription(
     # to auto-prolong subscription itself
 
 
+def mark_stuck_pending_payments_as_abandoned(older_than: timedelta = DEFAULT_ABANDON_PENDING_PAYMENTS_AFTER):
```

Comment:
This is not called automatically, right?

Reply:
What do you mean? It's just a task, one could configure celery beat or aws scheduler to run this task